src/util/loanSearch/queryParamUtils.js

```diff
@@ -202,6 +204,14 @@ export function updateQueryParams(loanSearchState, router, queryType) {
 
 	// Vue throws duplicate navigation exception when identical paths are pushed to the router
 	if (!doParamsMatch) {
-		router.push({ ...router.currentRoute, query: newParams, params: { noScroll: true, noAnalytics: true } });
+		router.push({ ...router.currentRoute, query: newParams, params: { noScroll: true, noAnalytics: true } })
+			.catch(e => {
+				const { isNavigationFailure, NavigationFailureType } = VueRouter;
```

Comment:
These weren't available from the passed `router` object?

Reply:
These aren't on the router instance but on the `VueRouter` module itself. These two items aren't exported individually from the Vue Router module. This pattern is pulled from the official Vue Router docs.